gradle/libs.versions.toml

```diff
@@ -35,12 +36,12 @@ androidx-workManager = "2.8.1"
 androidx-browser = "1.5.0"
 
 # Compose
-compose = "1.6.0-alpha05"
-compose-material = "1.6.0-alpha05"
+compose = "1.6.0-alpha07"
+compose-material = "1.6.0-alpha07"
 compose-activity = "1.7.2"
 compose-compiler = "1.5.2"
 compose-constraint = "1.0.1"
-compose-material3 = "1.2.0-alpha09"
+compose-material3 = "1.1.2"
```

Comment:
This was update by @Garzas to fix keyboard issue if I remember correctly, maybe downgrading it could show up the issue again